Tests/DatadogTests/Datadog/RUM/RUMMonitor/Scopes/RUMViewScopeTests.swift

```diff
@@ -416,14 +416,19 @@ class RUMViewScopeTests: XCTestCase {
             scope.process(command: RUMStopResourceCommand.mockWith(resourceKey: "/resource/1")),
             "The View should be kept alive as all its Resources haven't yet finished loading"
         )
+
+        var event = try XCTUnwrap(output.recordedEvents(ofType: RUMEvent<RUMViewEvent>.self).last)
+        XCTAssertTrue(event.model.view.isActive ?? false, "View should stay active")
+
         XCTAssertFalse(
             scope.process(command: RUMStopResourceWithErrorCommand.mockWithErrorMessage(resourceKey: "/resource/2")),
             "The View should stop as all its Resources finished loading"
         )
 
-        let event = try XCTUnwrap(output.recordedEvents(ofType: RUMEvent<RUMViewEvent>.self).last)
+        event = try XCTUnwrap(output.recordedEvents(ofType: RUMEvent<RUMViewEvent>.self).last)
         XCTAssertEqual(event.model.view.resource.count, 1, "View should record 1 successful Resource")
         XCTAssertEqual(event.model.view.error.count, 1, "View should record 1 error due to second Resource failure")
+        XCTAssertFalse(event.model.view.isActive ?? true, "View should be inactive")
     }
```

Comment:
This test looks good 👍, although we could also test that _"SDK doesn't send view updates to completed views (to ones which received their terminal, `view.isActive == false` update)"_. How about enhancing session-level validators in `RUMSessionMatcher` to also integrate this assertion? It already has the notion of `ViewVisit` aggregating all view updates in their time order and [even applying some preliminary `isActive` checks](https://github.com/DataDog/dd-sdk-ios/blob/e3468ed8dfb31a93c0f7b0831bfe9197ec15a53a/Tests/DatadogTests/Matchers/RUMSessionMatcher.swift#L214-L234).

We use `RUMSessionMatcher` in all integration and many unit tests, so this validation would be heavily applied.

Reply:
Yes, I can have a look!
So we would basically stop sending view updates once the view is inactive? 
What is the actual definition of an [`active`](https://github.com/DataDog/rum-events-format/blob/master/schemas/view-schema.json#L125) view?
I worry if BE start processing inactive view events in the future, we will loose data.

Reply:
> So we would basically stop sending view updates once the view is inactive?

This is exactly how I understand it 👍, but maybe @xgouchet can confirm this.

> What is the actual definition of an active view?

I will share more context on Slack.

Reply:
I need further modification to actually prevent from sending update events after a view is inactive, here it just sets `is_active: false`